modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the YAML view
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab, click the *Console (operator.openshift.io)* resource and see the *Details* page.
+. Click the *YAML* tab to open the editor and edit it as per your requirements. 
+.. For example, to disable a developer catalog type, insert the following snippet, which defines a list of disabled developer catalog resources.
+
++ 
+[source,yaml]
+----
+``yaml
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata: 
+  name: cluster
+  ...
+spec:
+  customization:
+    developerCatalog:
+      categories:
+      types:
+        state: Disabled
+        disabled:
+          - BuilderImage
+          - Devfile
+          - HelmChart
+----
+
+. Click *Save*.
+
+[Note]
+====
+By default, the developer catalog types are enabled in the Administrator view of the Web Console.
+====
+
+The following snippets can be used in the YAML editor for a variety of scenarios:
```

Comment:
I'd move the example used in the procedure above here and create a section dedicated to examples of snippets that the admin can use for various things.

Reply:
I would rather have this content under one section as I feel the flow of thought would be lost otherwise in a separate section. Here we are just providing some additional information that aids the main content, and keeping it separately would not be providing good value on it's own. This is just my suggestion.

Reply:
I noticed now that you use the `state: Disabled` snippet twice. Once as an example in the procedure and once in the list of snippets. I'd create a reference module called something like `Example YAML file changes` and feature all the snippets there. It won't break the flow of information. On the contrary, it will add to the clarity and scannability of the content.

Reply:
Fixed.